packages/mobile/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,11 +1,12 @@
 <network-security-config> 
   <domain-config cleartextTrafficPermitted="true">
     <domain includeSubdomains="true">localhost</domain>
+    <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
Good catch! I also just discovered this solution yesterday to unblock the e2e tests. Sorry I didn't share earlier.
But I believe we should remove this file and instead have this network_security_config be only in the `app/src/DEBUG/res/xml` folder so it only affects debug builds

Reply:
I will test that!